README.md

```diff
@@ -91,43 +103,65 @@ Note the configuration lines at the begining of the test:
 This is required for these tests to work.
 
 ### Example of a test
-
+The following test is based on dbt's jaffle-shop:
 ```jinja
 {{ config(tags=['unit-test']) }}
 
-{% call dbt_unit_testing.test('covid19_cases_per_day') %}
-  {% call dbt_unit_testing.mock_source('dbt_unit_testing_staging', 'covid19_stg') %}
-    select CAST('2021-05-05' as date) as day, '[{}]' as payload
-    union all
-    select CAST('2021-05-06' as date) as day, '[{"newCases": 20}]' as payload
+{% call dbt_unit_testing.test('customers', 'should sum order values to calculate customer_lifetime_value', options) %}
+  
+  {% call dbt_unit_testing.mock_ref ('stg_customers', options) %}
+    select 1 as customer_id

```

Comment:
does this have to be a number?

What are your thoughts on using descriptive values instead, eg:

select 'customer_1' as customer_id.

This will also indicated that this is a unit test, as this is not different from the on the seed data.

Reply:
The original jaffle_shop example has integers as ids, that's the reason, but at least if we can have the different orders of magnitude, it'll be more clear.